assemblyline/odm/models/submission.py

```diff
@@ -44,46 +49,47 @@ class ServiceSelection(odm.Model):
 }
 
 
-@odm.model(index=True, store=False, description="Submission Parameters")
+@odm.model(index=True, store=False, description="""Specifies the parameters set at the time of submission, such as whether to perform a deep scan, if the submission should generate an alert, and any specific service selections or settings.
+""")
 class SubmissionParams(odm.Model):
     classification = odm.Classification(default=Classification.UNRESTRICTED,
-                                        description="Original classification of the submission")
-    deep_scan = odm.Boolean(default=False, description="Should a deep scan be performed?")
-    description = odm.Text(store=True, copyto="__text__", description="Description of the submission")
-    generate_alert = odm.Boolean(default=False, description="Should this submission generate an alert?")
-    groups = odm.List(odm.Keyword(), default=[], description="List of groups related to this scan")
-    ignore_cache = odm.Boolean(default=False, description="Ignore the cached service results?")
+                                        description="The initial security classification for the submission, indicating its confidentiality.")
```

Comment:
"The original classification" means to me that it can change during the lifecycle of a submission.  In this case, `initial` kinda makes sense.  `Minimal` would mean that a submission would be, for example, at least unclass and maybe PB or Secret after some processing, but the classification value would not be changed.  

I don't know if the classification can change during the lifecycle of a submission.  If it can't, then in this case `Original` would mean that it is the classification as labeled at submission time.  

So in which situation are we in?

Reply:
So the classification provided at submission-time, is the classification of the submission document itself.